src/shared/rpc-contracts/app-rpcs/events.errors.ts

```
@@ -0,0 +1,142 @@
+    message: Schema.String,
+  },
+) {}
+
+export class EventRegistrationConflictError extends Schema.TaggedError<EventRegistrationConflictError>()(
+  'EventRegistrationConflictError',
+  {
+    message: Schema.String,
+  },
+) {}
+
+export class EventRegistrationInternalError extends Schema.TaggedError<EventRegistrationInternalError>()(
```

ESLint

[error] Use `new` when creating an error.

(unicorn/throw-new-error)